tests/components/bayesian/test_binary_sensor.py

```diff
@@ -128,6 +128,7 @@ async def test_sensor_numeric_state(hass):
         "sensor.test_monitored"
     ]
     assert abs(state.attributes.get("probability") - 0.14) < 0.01
+    # Calculated using bayes theorum where P(A) = 0.2, P(B|A) = 0.4 (as negative observation), P(B|notA) = 0.6
```

Comment:
Could you be more clear? 

Suggestion:

where:
A = sensor.test_binary being ON
B = sensor.test_monitored in the range [5, 10]

P(A|B) = P(B|A) * P(A) / P(B|A)*P(A) + P(B|~A)*P(~A)

(I'm also guessing you meant P(B|A) = 0.6 and P(B|~A) = 0.4 instead)

Reply:
Agreed my statement is ambiguous. It also doesn't help that `prob_given_false` and `prob_given_true` sum to 1 in this example which added to the ambiguity and hid the fact I was using  `1-prob_given_true`.

As per your suggestion I have now made this first test more verbose in the comments and reduced ambiguity. https://github.com/home-assistant/core/pull/67631/commits/904dbef8085c97ee0151fae5bb76844186b101a8